src/components/communities/MemberListPage.tsx

```diff
@@ -1,6 +1,12 @@
-import React from "react";
-import { StyleSheet, SectionList, ScrollView } from "react-native";

```

Comment:
is there a different way to add scrolling functionality? I placed this here for when members in the community is more than can be displayed on screen

Reply:
SectionList has a ScrollView built in according to the docs. So if the size of the list is too long it will automatically let you scroll

Reply:
oh, perfect, i must have missed that. looks good to me!